onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -287,7 +300,13 @@ public void onInfoWindowClick(Marker marker) {
 
         // Show trip details screen for the vehicle associated with this marker
         ObaTripStatus status = mMarkerData.getStatusFromMarker(marker);
-        TripDetailsActivity.start(mActivity, status.getActiveTripId());
+
+        String focusedStopId = mController.getFocusedStopId();
```

Comment:
You should check if `mController` is null before calling `mController.getFocusedStopId()`, to prevent potential NPEs.